### PR TITLE
Bugfix and reimplement Pictrs

### PR DESCRIPTION
**Pull request by Gildas-GH**
## Problem

The current package in master has three problems : 

- It is not up to date with upstream
- It is broken due to an error with Sharp
- It doesn't support image upload

## Solution

- The first problem cannot be solved as the PostgreSQL version available in Bullseye doesn't meet the requirements of Lemmy. We have to wait.
- The two other can be solved with #43 and a bugfix contained in this branch

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/change_url [hunk 1]
```diff
@@ -16,8 +16,9 @@ source /usr/share/yunohost/helpers
 #=================================================
 ynh_script_progression --message="Stopping a systemd service..." --weight=1
 
-ynh_systemd_action --service_name=$app --action="stop" --line_match="Stopped" --log_path=systemd
-ynh_systemd_action --service_name=$app-ui --action="stop" --line_match="Stopped" --log_path=systemd
+ynh_systemd_action --service_name=$app --action="stop" --line_match="Stopped" "/var/log/$app/$app.log"
```
Comment (ericgaspar) [inline scripts/change_url:19]:
```suggestion
ynh_systemd_action --service_name=$app --action="stop" --line_match="Stopped" --log_path="/var/log/$app/$app.log"
```

File: manifest.toml [hunk 2]
```diff
@@ -76,7 +82,7 @@ ram.runtime = "50M"
     inbox.protected = true
 
     [resources.apt]
-    packages = "postgresql espeak"
+    packages = "postgresql espeak rsync libfuse2 libimage-exiftool-perl ffmpeg"
```
Comment (ericgaspar) [inline manifest.toml:85]:
```suggestion
    packages = "postgresql, espeak, rsync, libfuse2, libimage-exiftool-perl, ffmpeg"
```

File: scripts/change_url (hunk 1 above)
Comment (ericgaspar) [reply scripts/change_url:19]: log_path are set to differently in upgrade script `--log_path=systemd`

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: scripts/change_url (hunk 1 above)
Comment (Gildas-GH) [reply scripts/change_url:19]: What's the best practice ? Write logs in the file ?

**Pull request merged**